test/framework/clusterctl/e2e_config.go

```diff
@@ -508,14 +508,9 @@ func (c *E2EConfig) AbsPaths(basePath string) {
 		provider := &c.Providers[i]
 		for j := range provider.Versions {
 			version := &provider.Versions[j]
-			if version.Type != URLSource && version.Value != "" {
-				if !filepath.IsAbs(version.Value) {
-					version.Value = filepath.Join(basePath, version.Value)
-				}
-			} else if version.Type == URLSource && version.Value != "" {
-				// Skip error, will be checked later when loading contents from URL
+			if version.Value != "" {
```

Comment:
Can we add a unit test for this function, so it will also act as a documentation of all the combinations we support?